### DISCUSSION TITLE
Github Pages: Incorrect "NotServedByPagesError"

### DISCUSSION THREAD

**Discussion by rail5** [community/community#73042] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

I'm having some trouble with Github Pages

[My site](https://rail5.org)'s SSL certificate expired on Oct. 24th and was not automatically renewed by Github. When I go to the Pages tab in the repo to try to fix this, it claims (incorrectly) that my domain's DNS is not properly configured (and therefore that it can't renew the SSL certificate)

The DNS records are not only properly configured but also haven't changed in several months, so there's no possibility of a "change propagation delay" issue. Github also didn't used to have any issue with these same (unchanged) records

Any ideas on whether there's anything I could do to fix this? I'm completely lost on this one

**Comment by DivyamGupta3000** [comment]


**Reply by rail5** [reply]
Please don't use ChatGPT to try to answer questions. If any of us want an answer from ChatGPT, we'll ask ChatGPT
